modules/ROOT/pages/aurads/managing-instances/advanced-metrics.adoc

```diff
@@ -1,8 +1,6 @@
 [[aura-monitoring]]
 = Advanced metrics
 
-label:AuraDS-Enterprise[]
```

Comment:
As we don't have a free tier of AuraDS, all AuraDS users will have access to AAM, so removing the label.